### PR TITLE
ACMS-3416: Remove amplitude log request from ACMS Common module.

### PR DESCRIPTION
**Pull request by rajeshreeputra**
**Motivation**

Fixes #NNN

**Proposed changes**

**Alternatives considered**

**Testing steps**

**Merge requirements**
- [ ] _Major change_, _Minor change_, _Bug_, _Enhancement_, and/or _Chore_ label applied
- [ ] Manual testing by a reviewer

### CODE REVIEW

**Review by rajeshreeputra — COMMENTED** [review]
LG

**Review by apathak18 — APPROVED** [review]

**Review by chandan-singh7929 — APPROVED** [review]

File: modules/acquia_cms_common/composer.json [hunk 1]
```diff
@@ -81,6 +81,9 @@
             },
             "drupal/workbench_email": {
                 "3408131 - Route 'entity.workbench_email_template.canonical' does not exist.": "https://git.drupalcode.org/project/workbench_email/-/merge_requests/17.patch"
+            },
+            "drupal/acquia_connector": {
```
Comment (vishalkhode1) [inline modules/acquia_cms_common/composer.json:85]: @rajeshreeputra Why we added this patch ? What this is doing ?

**Review by vishalkhode1 — CHANGES_REQUESTED** [review]

File: modules/acquia_cms_common/composer.json (hunk 1 above)
Comment (deepakmishra2) [reply modules/acquia_cms_common/composer.json:85]: @vishalkhode1 this patch has been added to remove telemetry from acquia_connector module and i feel we can move this to our root composer.json. what do you say @vishalkhode1 ?

Comment (deepakmishra2) [reply modules/acquia_cms_common/composer.json]: done

**Review by vishalkhode1 — APPROVED** [review]

**Pull request merged**